pootle/templates/browser/_actions.html

```diff
@@ -1,25 +1,25 @@
 {% load i18n %}
 <ul>
   <li id="js-action-continue">
-    <a class="continue-translation" href="{{ url_action_continue }}">
+    <a class="continue-translation"{% if url_action_continue %} href="{{ url_action_continue }}"{% endif %}>
```

Comment:
My understanding this is making the button not have a href. Yet it continues to behave like it is clickable?

Can we make sure that we turn the button into an info only display and make sure it clearly doesn't appear to be clickable.